usagov/vote-gov-drupal
# VOTE-2959: Remove registration intro fields

## Jira ticket

[VOTE-2959](https://cm-jira.usa.gov/browse/VOTE-2959)

## Description

Insert a brief summary of the changes included in this PR and any additional information or context which may help the reviewer.

## Deployment and testing

### QA/Testing instructions

1. lando retune
3. Navigate to http://vote-gov.lndo.site/block/add/state_display_content?destination=/admin/content/block
4. Confirm no fields exist for registration intro
5. Navigate to http://vote-gov.lndo.site/admin/structure/block-content/manage/state_display_content/fields
6. Confirm no fields exist for registration intro

## Checklist for the Developer




## Checklist for the Peer Reviewers
